source/meta.tex

```diff
@@ -2066,7 +2066,6 @@
 \tcode{\placeholdernc{COMMON-REF}(A, B)} is ill-formed.
 
 \pnum
-Note A:
```

Comment:
I think these changes are definite improvements, I've wanted to be rid of these normative "notes" for a long  time.